cypress/integration/dryRunner/discardNoPlay.js

```diff
@@ -0,0 +1,35 @@
+import s from './selectors'
+
+const DECK_ID =
+  'NU41LDVOOCw1TjEyLDVOMTQsNU4xNiw1TjE5LDVOMjIsNU4zMCw1VzksNVcxNiw1TjQyLDVONzI='
+
+describe('Dry-runner — Don’t play discarded cards', () => {
```

Comment:
I would put this one in basics as it seems to be an extension of the play/cycle tests.